code/Controllers/CMSMain.php

```diff
@@ -1089,11 +1090,65 @@ public function Breadcrumbs($unlinked = false)
             ]));
         }
 
+        $badge = null;
+        $status = $this->getStatusFlag($record);
+
+        if (count($status) > 0) {
+            $flags = '';
+            foreach ($status as $flag => $flagData) {
+                $flags .= sprintf(
+                    '<span class="badge version-status version-status--%s" %s>%s</span>',
+                    'status-' . Convert::raw2xml($flag),
+                    (isset($flagData['title'])) ? sprintf(' title="%s"', Convert::raw2xml($flagData['title'])) : '',
+                    Convert::raw2xml($flagData['text'])
+                );
+            }
+            $badge = DBField::create_field('HTMLFragment', $flags);
+        }
+
+        $this->extend('updateBadge', $badge);
+
+        if ($badge) {
+            /** @var ArrayData $lastItem */
+            $lastItem = $items->last();
+            $lastItem->setField('Extra', $badge);
+        }
+
         $this->extend('updateBreadcrumbs', $items);
 
         return $items;
     }
 
+    private function getStatusFlag($record)
+    {
+        $flags = [];
+        if ($record->isOnLiveOnly()) {
+            $flags['removedfromdraft'] = [
+                'text' => _t(__CLASS__.'.ONLIVEONLYSHORT', 'On live only'),
```

Comment:
```suggestion
                'text' => _t(__CLASS__ . '.ON_LIVE_ONLY_SHORT', 'On live only'),
```
Need spaces around the `.` used for concatenation, and add underscores between words

Do this for all added translations

Reply:
`ONLIVEONLYSHORT`, `ARCHIVEDPAGESHORT`, `MODIFIEDONDRAFTSHORT` already exist in `yml` files for `CMSMain` class.  And `ADDEDTODRAFTSHORT` I've added as new for `CMSMain` class.

Reply:
Updated. I left `ONLIVEONLYSHORT` as is since we already have this key in lang files.